app/src/main/res/values/strings-preferences.xml

```diff
@@ -16,6 +16,15 @@
     <string name="pref_key__clear_cache" translatable="false">pref_key_clear_cache</string>
     <string name="pref_key__chrome_custom_tabs_enabled" translatable="false">pref_key__chrome_custom_tabs_enabled</string>
 
+    <!-- Themes -->
+    <string name="pref_key__primary_color" translatable="false">pref_key_primary_color</string> <!--onPreferenceClick-->
```

Comment:
Remove comment, will go into a **seperate** line in every language:
```
<!--onPreferenceClick-->
<string name="pref_key__primary_color_shade" translatable="false">pref_key_primary_color_shade</string>
<string name="pref_key__accent_color" translatable="false">pref_key_accent_color</string>
<!--onPreferenceClick-->
<string name="pref_key__accent_color_base" translatable="false">pref_key_accent_color_base</string>
```
Maybe adjust the key name instead